analyses/code_to_review.R

```diff
@@ -0,0 +1,86 @@
+target_feature_value <- params$added_feature
+target_feature_shap <- params$added_feature_shap
+nbhd <- ccao::nbhd_shp
+
+# Selecting and joining relevant data
+card_individual <- shap %>%
+  select(meta_pin, meta_card_num, pred_card_shap_baseline_fmv,
+         {{ target_feature_value }}) %>%
+  rename(!!target_feature_shap := !!target_feature_value) %>%
+  inner_join(assessment_card, by = c("meta_pin", "meta_card_num")) %>%
+  inner_join(
+    assessment_data %>%
+      select(meta_pin, meta_card_num, meta_nbhd_code,
+             loc_longitude, loc_latitude, meta_township_name),
+    by = c("meta_pin", "meta_card_num")
+  )
+
+
+# Summarizing data by neighborhood code
+card_nbhd <- card_individual %>%
+  group_by(meta_nbhd_code) %>%
+  summarize(
+    avg_target_feature_shap =
+      mean(!!sym({{target_feature_shap}}), na.rm = TRUE),
+    avg_target_feature_shap_abs =
+      mean(abs(!!sym({{target_feature_shap}})), na.rm = TRUE),
+    avg_pred_card_shap_baseline_fmv =
+      mean(pred_card_shap_baseline_fmv, na.rm = TRUE)
+  ) %>%
+  ungroup()
+
+
+pin_individual <- assessment_pin %>%
+  select(meta_pin, pred_pin_final_fmv, pred_pin_initial_fmv) %>%
```

Comment:
At the moment, the only reason to really include the pin level data is for fair market values or possibly sales if we want to include them. Do we want to include this information in the leaflet mapping, or just do things on the card level?